.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: ci
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This means we'd only trigger the CI on PRs against the master branch, correct?

If so, could we relax this restriction so that we can "chain" PRs if needed and still run CI?